### PR TITLE
WT-6546 Update fast truncate to use newest start durable ts

### PR DESCRIPTION
**Pull request by luke-pearson**

### CODE REVIEW

File: test/suite/test_truncate05.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        self.session.begin_transaction('read_timestamp=' + timestamp_str(2))
+
+        # Truncate from key 1 to 1000.
+        start = self.session.open_cursor(uri, None)
+        start.set_key(1)
+        end = self.session.open_cursor(uri, None)
+        end.set_key(1000)
+
+        # Call the truncate and expect failure.
+        self.assertRaisesException(
```
Comment (quchenhao) [inline test/suite/test_truncate05.py:82]: What is the type of exception raised here?

Comment (luke-pearson) [reply test/suite/test_truncate05.py:81]: Its a WT_ROLLBACK error which translates to wiredtiger.WiredTigerError in our python test suite.

Comment (quchenhao) [reply test/suite/test_truncate05.py:81]: Can we check it's WT_ROLLBACK in the error message?

**Review by quchenhao — APPROVED** [review]
This is a step closer to the end goal. LGTM!

File: test/suite/test_truncate05.py (hunk 1 above)
Comment (luke-pearson) [reply test/suite/test_truncate05.py:81]: This error doesn't get return by WiredTiger with a message, I tried converting it to `assertRaisesWithMessage` to test.

**Review by bvpvamsikrishna — APPROVED** [review]
lgtm

**Pull request merged**